pkg/loki/loki.go

```diff
@@ -505,6 +503,7 @@ func (t *Loki) bindConfigEndpoint(opts RunOpts) {
 		configEndpointHandlerFn = opts.CustomConfigEndpointHandlerFn
 	}
 	t.Server.HTTP.Path("/config").Methods("GET").HandlerFunc(configEndpointHandlerFn)
+	t.Server.HTTP.Path("/config/tenant/{tenant}/limits").Methods("GET").HandlerFunc(t.tenantLimitsHandler())
```

Comment:
I'm not sure if we have a root prefix for paths like this? given that this is now going to be something relied upon by other services I think the path should have a version in it, if that's not already provided by an additional prefix somewhere else